ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1120,6 +1120,17 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
             }
             break;
 
+        case MAV_CMD_DO_GO_AROUND: {
+            float go_around_alt_m = 3.0f;
```

Comment:
Why the hardcoded 3m ? Is it documented in mavlink ?

Reply:
It is a default value. Mavlink document param#1 as altitude but do not specify behavior if param#1 is 0. Mission planner send MAV_CMD_DO_GO_AROUND with param#1 0. I give it a default value (3m) so that it will work with mission planner